recrypt/recrypt.c

```diff
@@ -36,6 +36,7 @@ int tarsnap_opt_noisy_warnings = 0;
 int tarsnap_opt_humanize_numbers = 0;
 uint64_t tarsnap_opt_checkpointbytes = (uint64_t)(-1);
 uint64_t tarsnap_opt_maxbytesout = (uint64_t)(-1);
+uint64_t tarsnap_opt_uploadprogressbytes = (uint64_t)(-1);
```

Comment:
Not certain if you want underscores in the second part of this name.  Sometimes you do:
```
extern int tarsnap_opt_retry_forever;
extern int tarsnap_opt_humanize_numbers;
```
but sometimes you don't:
```
--checkpoint-bytes
extern uint64_t tarsnap_opt_checkpointbytes;
```

In the end, I decided that `--upload-progress-bytes` was more similar to `--checkpoint-bytes` then `--retry-forever`.

Reply:
That spelling is fine.